Source/WebKit/WebProcess/GPU/graphics/WebGPU/RemoteAdapterProxy.h

```diff
@@ -76,8 +76,13 @@ class RemoteAdapterProxy final : public PAL::WebGPU::Adapter {
     {
         return root().streamClientConnection().sendSync(WTFMove(message), backing(), defaultSendTimeout);
     }
+    template<typename T, typename C>
+    WARN_UNUSED_RETURN IPC::StreamClientConnection::AsyncReplyID sendWithAsyncReply(T&& message, C&& completionHandler)
```

Comment:
Can we just add this everywhere that needs it in a single standalone patch, instead of adding it lazily? Every async call will need it.